zcash-hsmbuilder/src/txbuilder.rs

```diff
@@ -465,7 +488,10 @@ impl<P: consensus::Parameters, R: RngCore + CryptoRng>
 
                 // Update progress and send a notification on the channel
                 progress += 1;
-                progress_notifier.as_ref().map(|tx| tx.send(progress));
+                if let Some(sender) = progress_notifier.as_ref() {
+                    // If the send fails, we should ignore the error, not crash.
+                    sender.send(Progress::new(progress, None)).unwrap_or(());
```

Comment:
if you want to ignore the error just `let _ = ...` instead, it's the conventional way to ignore erros